Scripts/Runtime/Util/Random/RandomExtensions.cs

```diff
@@ -0,0 +1,38 @@
+using System.Runtime.CompilerServices;
+using Unity.Jobs.LowLevel.Unsafe;
+using Unity.Mathematics;
+
+namespace Anvil.Unity.DOTS.Util
```

Comment:
Should we put this in the `...DOTS.Random` namespace? I could see other random implementations in the future.

Reply:
I tried but it then messes with the imports everywhere because `Random` is now a namespace. 

If we do have other random implementations in the future I think we'll want to rename these classes anyway and we can change it then.